sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -161,6 +161,8 @@ class Analyzer(
       HandleNullInputsForUDF),
     Batch("FixNullability", Once,
       FixNullability),
+    Batch("ResolveAggAliasInGroupBy", Once,
+      ResolveAggAliasInGroupBy),
```

Comment:
When I moved `ResolveOrdinalInOrderByAndGroupBy` outside `Resolution` batch, I hit test errors. So, I'm looking into the reason..